platformer/src/processing/MissionPossible/Sprite.pde

```diff
@@ -0,0 +1,18 @@
+public abstract class Sprite{
+  int xPos;
+  int yPos;
+  float speed;
+  float yAcceleration;
+  float xAcceleration;
+  int spriteWidth;
+  int spriteHeight;
+  int spriteLayer;
+  int health; // Move to PlayerControlledSprite if only player has health
+  boolean onGround;
+  PImage image;
+  // TODO direction - type?
+  // TODO damage - type?
+  // TODO collision - type?
```

Comment:
Having thought about it a bit more, given that we have x and y Acceleration, probably makes sense to remove `direction` and have `xSpeed` and `ySpeed` instead?
`damage` would probably be a float, `collision` a boolean?

Reply:
Have made these changes - probably need to think about whether we need all of lives, health and damage.